cli/tools/repl.rs

```diff
@@ -721,6 +736,14 @@ pub async fn run(
   let history_file_path = program_state.dir.root.join("deno_history.txt");
   let editor = ReplEditor::new(helper, history_file_path);
 
+  if let Some(eval) = maybe_eval {
+    let output = repl_session.evaluate_line_and_get_output(&eval).await?;
+    // only output errors
+    if let EvaluationOutput::Error(error_text) = output {
+      println!("error in --eval flag. {}", error_text);
+    }
+  }
```

Comment:
It was super easy to implement this thanks to the recent refactorings in this code 😀